requirements.txt

```diff
@@ -1,47 +1,47 @@
 # Autogenerated by 'make bs'
 # If the format has changed, update MIN_PIPENV_VERSION in Makefile
 -i https://pypi.org/simple
-bracex==2.3.post1 ; python_version >= '3.7'
```

Comment:
I'm working on figuring out what the new MIN_PIPENV_VERSION should be

Reply:
Not having luck with this. If my env is the only one making this format change, I can just revert this file manually.